src/patternfly/components/DualListSelector/dual-list-selector-tools.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-dual-list-selector__tools{{#if dual-list-selector-tools--modifier}} {{dual-list-selector-tools--modifier}}{{/if}}"
```

Comment:
should this be `__toolbar`

Reply:
I don't think so, since toolbar is a component, and this isn't for a toolbar. `<Page>` uses `header-tools` in a similar way.